app/scripts/views/object/display-data.js

```diff
@@ -0,0 +1,40 @@
+import React, {PropTypes} from 'react'
+
+import i18n from '../../utils/i18n'
+import RawData from './raw-data'
+
+const DisplayData = ({data}) => {
+  const size = data && data.length - 2
+
+  if (!size) {
```

Comment:
Size is not the best const name. I would take the const out entirely, and just use `(data && data...)`